trlx/utils/modeling.py

```diff
@@ -126,6 +155,11 @@ def hf_get_causal_hidden_layers(model: nn.Module) -> Tuple[nn.Module]:
     return findattr(model, hidden_layers_attrs)
 
 
+def hf_decoder_hidden_layers(model: nn.Module) -> Tuple[nn.Module]:
+    hidden_layers_attrs = "decoder.block"
+    return findattr(model, hidden_layers_attrs)
```

Comment:
This is unused. Is this intentional? If so, let's remove it.

Reply:
I have a plan for when we have options to freeze the encoder, and keep the decoder but it is ok to remove it now.

Reply:
Oooo that sounds interesting. Thanks for bringing it up!